crates/rpc/src/lib.rs

```diff
@@ -1,14 +1,52 @@
-pub fn add(left: u64, right: u64) -> u64 {
-    left + right
+use std::{net::SocketAddr, sync::Arc};
+
+use config::ServerConfig;
+use routes::get_routes;
+use utils::{chain::BeaconChain, error::handle_rejection};
+use warp::Filter;
+
+pub mod config;
+pub mod handlers;
+pub mod routes;
+pub mod types;
+pub mod utils;
+
+/// Start the Beacon API server.
+pub async fn start_server(ctx: Arc<BeaconChain>, server_config: ServerConfig) {
+    let addr: SocketAddr = format!("{}:{}", server_config.http_address, server_config.http_port)
+        .parse()
+        .unwrap();
```

Comment:
```suggestion
        .unwrap();
```

Can we not use unwrap and instead return an error